src/swap/changelly.js

```diff
@@ -114,18 +116,30 @@ async function getAddress(
     : addressInfo.publicAddress
 }
 
-function checkReply(reply: Object, request: EdgeSwapRequest) {
+function checkReply(reply: Object, request: EdgeSwapRequest, log: EdgeLog) {
   if (reply.error != null) {
     if (
       reply.error.code === -32602 ||
       /Invalid currency:/.test(reply.error.message)
     ) {
+      log.warn(`${pluginId} SwapCurrencyError ${JSON.stringify(request)}`)
```

Comment:
We should probably not log these errors - the core already has centralized logging logic for dealing with all the swaps as a group. If we honestly want all errors logged, we should simply adjust the core, not all the plugins.